src/errorHandle/errorMessage.ts

```diff
@@ -0,0 +1,11 @@
+export const getErrorMessage = (error: unknown): string => {
```

Comment:
Weird way of obtaining the error, can we make sure this is the best approach to accomplish this?